cinema/views.py

```diff
@@ -0,0 +1,44 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+from django.shortcuts import get_object_or_404
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+        
+        if serializer.is_valid():

```

Comment:
use serializer.is_valid(raise_exception=True) 
This way you don't need to perform a check and return 400, it will be done automatically

Do the same in movie_detail